### PR TITLE
Fixes issue #21 for enabling terraform destroy and also specifying final snapshot for rds

### PR DESCRIPTION
**Pull request by miketwenty1**
Creating db_final_snapshot_identifier variable and including it in rds.tf. This will make it possible to use the terraform destroy command against this module (#21)

### CODE REVIEW

**Comment by miketwenty1** [comment]
The second commit that adds the depends on for rds and elasti cache is needed.  `terraform destroy` will fail to wait for these data store resources to be completely destroyed before trying to delete the associated parameter groups.

File: variables.tf [hunk 1]
```diff
@@ -515,3 +515,8 @@ variable "deck_version" {
   default = "1.0.0"
 }
 
+variable "db_final_snapshot_identifier" {
+  description = "identifier"
```
Comment (DennoVonDiesel) [inline variables.tf]: Can you provide a more complete description here? Thanks for the contribution!

Comment (miketwenty1) [reply variables.tf]: @DennoVonDiesel done.

**Comment by miketwenty1** [comment]
fixed some other stuff that was failing the workflow that @rlnchowrlnchow put in.

**Comment by miketwenty1** [comment]
fixed some other stuff that was failing the workflow that @rlnchow put in.

**Comment by miketwenty1** [comment]
@DennoVonDiesel are you going to bump the git tag?

**Comment by DennoVonDiesel** [comment]
> @DennoVonDiesel are you going to bump the git tag?

I just did with v3.4, thanks!

**Pull request merged**